lib/Site.js

```diff
@@ -522,6 +522,12 @@ Site.prototype.generatePages = function () {
   });
   return new Promise((resolve, reject) => {
     Promise.all(processingFiles)
+      .then((res) => {
```

Comment:
The `res` doesn't seem to be necessary (the result is an undefined array, and is not used by subsequent code), so it can be removed (just `.then(() => {`)